migrations/witnet.requests.json

```diff
@@ -101,6 +101,12 @@
         "quote": "USD",
         "decimals": 6
     },
+    "ElonUsdtPrice": {
+        "bytecode": "0x0a8104124a0801122f68747470733a2f2f646174612e676174656170692e696f2f617069322f312f7469636b65722f656c6f6e5f757364741a15841877821864646c6173748218571a000f4240185b126a0801124668747470733a2f2f6170692e6b75636f696e2e636f6d2f6170692f76312f6d61726b65742f6f72646572626f6f6b2f6c6576656c313f73796d626f6c3d454c4f4e2d555344541a1e85187782186664646174618218646570726963658218571a000f4240185b125e0801123a68747470733a2f2f6170692e68756f62692e70726f2f6d61726b65742f64657461696c2f6d65726765643f73796d626f6c3d656c6f6e757364741a1e851877821866647469636b82186465636c6f73658218571a000f4240185b12660801123f68747470733a2f2f7777772e6d6578632e636f6d2f6f70656e2f6170692f76322f6d61726b65742f7469636b65723f73796d626f6c3d454c4f4e5f555344541a21861877821861646461746182181800821864646c6173748218571a000f4240185b12610801123a68747470733a2f2f7777772e6f6b65782e636f6d2f6170692f76352f6d61726b65742f7469636b65723f696e737449643d454c4f4e2d555344541a21861877821861646461746182181800821864646c6173748218571a000f4240185b1a0d0a0908051205fa3fc000001003220d0a0908051205fa40200000100310c0843d180a20c0843d28333080e497d012",
```

Comment:
This bytecode is not right, most probably simply outdated. It multiplies by 10 ** 6 instead of 10 ** 9.